tests/e2e/README.md

```diff
@@ -96,27 +97,33 @@ Setup Wizard e2e test (located in `activate-and-setup` directory) will run befor
 
 - Run `npm run build:core`
 
-- Run the following command to build the test site using Docker: `npm run docker:up` and watch the site being built. Note that it may take a few minutes the first time you do that. The process is considered completed when the messages letting you know that WordPress was installed, WooCommerce was activated and users created will be displayed:
+- Run `npm run docker:up` - it will build the test site using Docker.  
+
+- Run `docker ps` - to confirm that the Docker containers were built and running. You should see the log that looks similar to below indicating that everything had been built as expected:
 
 ```
-wordpress-cli_1             | Success: WordPress installed successfully.
-wordpress-cli_1             | Plugin 'woocommerce' activated.
-wordpress-cli_1             | Success: Activated 1 of 1 plugins.
-wordpress-cli_1             | Success: Created user 2.
-woocommerce_wordpress-cli_1 exited with code 0
-woocommerce_wordpress-cli_1 exited with code 0
+CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                  NAMES
+c380e1964506        env_wordpress-cli   "entrypoint.sh"          7 seconds ago       Up 5 seconds                               woocommerce_wordpress-cli
+2ab8e8439e9f        wordpress:5.5.1     "docker-entrypoint.s…"   8 seconds ago       Up 7 seconds        0.0.0.0:8084->80/tcp   woocommerce_wordpress-www
+4c1e3f2a49db        mariadb:10.5.5      "docker-entrypoint.s…"   10 seconds ago      Up 8 seconds        3306/tcp               woocommerce_db
 ```
 
-For more Docker commands, scroll down to [Docker basics](#docker-basics).
+Note that by default, Docker will download the latest images available for WordPress, PHP and MariaDB. In the example above, you can see that WordPress 5.5.1 and MariaDB 10.5.5 were used. 
 
-- Open new terminal window and `cd` to the current branch again.
+See [How to run tests using custom WordPress, PHP and MariaDV versions](#how-to-run-tests-using-custom-wordpress,-php-and-mariadv-versions) if you'd like to use different versions.  
 
-- Run the following command to make sure the containers were built and running: `docker ps`. You should see the 2 following containers: 
+- Navigate to `http://localhost:8084/`
 
-- `woocommerce_wordpress-woocommerce-dev`
-- `mariadb:10.4`
+If everything went well, you should be able to access the site. If you changed the port to something other than `8084`, use the appropriate port to access your site. 
 
-- Navigate to `http://localhost:8084/`. If everything went well, you should be able to access the site.  
+As noted in [Test Variables](#test-variables) section, use the following Admin user details to login:
+
+```
+Username: admin
+PW: password
+```
+
+- Run `npm run docker:down` when you are done with running e2e tests or when making any changes to test suite.
```

Comment:
It might be worth noting in this section that running docker:down then docker:up re-initializes the test container.

Reply:
Added note as per your suggestion, thanks @rrennick!